README.md

```diff
@@ -2265,6 +2265,7 @@ Manages configuration of ITD (Intelligent Traffic Director) service.
 
 | Property | Caveat Description |
 |:--------|:-------------|
+| `Entire` <br> `provider` <br> | For this provider to work, CLI commands need to be executed <br> via command_config prior to invoking the manifest. They are <br> 'feature pbr' <br> 'feature sla sender' <br> 'feature sla responder' <br> 'ip sla responder' |

```

Comment:
Please test that you can literally cut and paste this into a site.pp file and apply to make it easier for users.